sympy/solvers/solvers.py

```diff
@@ -586,50 +603,49 @@ def solve(f, *symbols, **flags):
         >>> solve(x**4 - 1, x, set=True)
         ([x], {(-1,), (1,), (-I,), (I,)})
 
-    * single expression with no symbol that is in the expression
+    Single expression with no *symbol* that is in the expression:
 
         >>> solve(3, x)
         []
         >>> solve(x - 3, y)
         []
 
-    * single expression with no symbol given
+    Single expression with no *symbol* given. In this case, all free symbols
+    will be selected as potential symbols to solve for. If the equation is
+    univariate then a list of solutions is returned; otherwise — as is the case
+    when symbols are given as an iterable of length > 1 — a list of mappings
+    will be returned:
```

Comment:
The text `length > 1 — a` is confusing here as it looks like a mathematical fragment...

Reply:
Is it supposed to be formatted as math? Or can you make a suggestion of how to complete it?

Reply:
`>` should be replaced with the words "greater than"